projects/dsp-ui/src/lib/action/components/string-literal-input/string-literal-input.component.ts

```diff
@@ -210,6 +214,9 @@ export class StringLiteralInputComponent implements OnInit {
         if (!value) {
             value = '';
         }
+        if (!this.form) {
+            return;
+        }
```

Comment:
Is this still necessary? I think we added it because of the wrong implementation of `ngOnChanges`?

Reply:
`updateFormField` will be called from `ngOnChanges` which will be called before the form gets created in `ngOnInit` so I think it's necessary still